app/src/ui/changes/changes-list.tsx

```diff
@@ -158,11 +175,112 @@ export class ChangesList extends React.Component<IChangesListProps, {}> {
     showContextualMenu(items)
   }
 
-  public render() {
+  // ripped from https://stackoverflow.com/a/9229821/1964166
+  private static Uniquify(array: string[]) {
```

Comment:
Where should we put such a method? the CI bots want it at the top of the class, but i believe it's not the right file for this. Any suggestions?

Reply:
@icosamuel if this is not dependent on state inside the class, this can live as a plain `function` outside the component in this file until we've had a better chance to review the changes

Reply:
@shiftkey Thanks for the fast feedback! I decided to integrate it to the code since there was no reuse